src/api/app/controllers/attribute_controller.rb

```diff
@@ -235,27 +233,19 @@ def cmd_attribute
     req = ActiveXML::Node.new(request.raw_post)
 
     # checks
-    if params[:attribute]
-      unless User.current.can_create_attribute_in? @attribute_container, namespace: params[:namespace], name: params[:name]
+    req.each('attribute') do |attr|
+      begin
+        can_create = User.current.can_create_attribute_in? @attribute_container, namespace: attr.value('namespace'), name: attr.value('name')
+      rescue ArgumentError => e
```

Comment:
I dont think its necessary to rescue this error here. `@attribute_container` will either be a `Project` or a `Package` from here so in the current implementation that error will never get raised.

https://github.com/hennevogel/open-build-service/blob/754a7cf42bf27671544a3a26cba05e6e5d4a38bb/src/api/app/controllers/attribute_controller.rb#L265